Scripts/script-IsMaliciousIndicatorFound.yml

```diff
@@ -27,10 +27,14 @@ script: |-
   if (ips && (!Array.isArray(ips) || ips.length > 0)) {
       return 'yes';
   }
-  var indicatorsRes = executeCommand("findIndicators", {"query":"(reputation:Bad or reputation:Suspicious)  and investigationIDs:"+investigation.id,"size":1});
+  var conditionExtra = "";
+  if (args.includeSuspicious == 'yes') {
+      conditionExtra = " or reputation:Suspicious";
+  }
+  var indicatorsRes = executeCommand("findIndicators", {"query":"(reputation:Bad"+conditionExtra+") and investigationIDs:"+investigation.id,"size":1});
```

Comment:
Maybe add extraConditions argument to allow for custom queries like (reputation:Bad or reputation:Suspicious or myCustomField:true)